starter/cms-payload/README.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Payload with Next.js Starter
+slug: cms-payload
+description: Next.js app with Payload, all in one repo.
+framework: Next.js
+useCase:
+  - Starter
+css: CSS Modules
+deployUrl: https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/starter/cms-payload&project-name=cms-payload&repository-name=cms-payload
+demoUrl: https://cms-payload.vercel.app
+relatedTemplates:
```

Comment:
I believe this is required, could we add some? Maybe just the Next.js boilerplate starter.